src/modules/events-app.module/module.html

```diff
@@ -10,7 +10,7 @@
 
 <div class="event-registration">
   <noscript>You need to enable JavaScript to run this app.</noscript>
-  <div id="root" data-portal-id="{{ portal_id }}"></div>
+  <div id="root" data-portal-id="{{ portal_id }}" data-slug="{{ slug }}"></div>
```

Comment:
We may want to consider a JavaScript based solution and also reconsider how we're approaching `IMAGES` so that we have a cleaner approach for passing module data to the JavaScript stuff.